lib/beaker/ssh_connection.rb

```diff
@@ -267,6 +267,7 @@ def scp_to source, target, options = {}
         logger.warn "#{e.class} error in scp'ing. Forcing the connection to close, which should " <<
           "raise an error."
         close
+        raise "#{e}\n#{e.backtrace}"
```

Comment:
@tvpartytonight I added the backtrace

Reply:
Oh, I was just thinking about printing the backtrace, not necessarily raising it itself.

[Here](https://github.com/puppetlabs/beaker/commit/e1c2beaaa5413587c8d34614e73efad603e7eab0) it looks the comment was relying on some implementation to raise the error in the underlying library. So, I'm curious if that has changed.

Reply:
yeah, I looked at that commit, but it was unclear to me what exactly that was referring to, since the `close` methods in this repo don't seem to do that. but I guess maybe you're saying that would from one of our dependencies?

Reply:
still doesn't seem like it ever raised errors?
https://github.com/net-ssh/net-ssh/blob/master/lib/net/ssh/connection/session.rb#L122

Reply:
oh, I didn't even see those comments 😅 I just skimmed right past the tests lol